src/app/rpmostree-builtin-status.cxx

```diff
@@ -302,6 +344,20 @@ print_daemon_state (RPMOSTreeSysroot *sysroot_proxy,
                  update_driver_name, update_driver_sd_unit);
       else
         g_print ("AutomaticUpdates: driven by %s\n", update_driver_name);
+      
+      /* only try to get unit's StatusText if we're on the system bus and if 
+       * `update_driver_sd_unit` is a systemd service unit. */
+      if (bus_type == G_BUS_TYPE_SYSTEM && g_str_has_suffix (update_driver_sd_unit, ".service"))
+        {
+          g_autofree const char *update_driver_status = NULL;
+          if (!get_update_driver_sd_status(sysroot_proxy, update_driver_sd_unit,
+                                           &update_driver_status, cancellable, error))
+            return FALSE;
+          if (update_driver_status && update_driver_status[0] != '\0')
+            g_print ("UpdateDriverStatus: %s\n", update_driver_status);
```

Comment:
sounds good :)

Reply:
This will probably end up with a lot of parenthesis in the normal happy path, for example
```
active (polling for ... (last checked ...))
```

What about keeping the active case as is, and just plugging-in the `<inactive>` detection?

Reply:
After changing from `Status` to `State`, because `AutomaticUpdates`' "child" `State` is just two lines below `rpm-ostreed`'s state, it looked a bit ambiguous and confusing. I opted for the slightly longer `DriverState` instead.
```
$ rpm-ostree status
State: idle
AutomaticUpdates: driven by Zincati
  DriverState: active, polling for ... (last checked ...)
Deployments:
* ostree://f30001a5d7f7c3c8e8dd409fe4ad68d052f715617644f94772b27b5f7cd12b4e
                   Version: fastbuild-1612240746 (2021-02-02T04:39:14Z)

  ostree://fedora:fedora/x86_64/coreos/testing-devel
                   Version: 33.20210129.dev.0 (2021-01-29T23:39:35Z)
                    Commit: be9444b56a6aaa7c4da71dd122b366355775c9ffc6733f1b375e832059d329c1
              GPGSignature: (unsigned)
```

Reply:
@lucab How does the above output look?

Reply:
I think I'd vote for
`AutomaticUpdateDriver: zincati.service`
over `AutomaticUpdates: driven by ...`.

Reply:
In my mind `AutomaticUpdates:` refers to rpm-ostree's built in support, which...we should probably soft-deprecate as in no new major features etc. and point people at zincati and otehr tools.